src/test/java/io/xpring/payid/PayIDClientTest.java

```diff
@@ -0,0 +1,107 @@
+package io.xpring.payid;
+
+import com.github.tomakehurst.wiremock.junit.WireMockRule;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+
+import static com.github.tomakehurst.wiremock.core.WireMockConfiguration.wireMockConfig;
+import static org.junit.Assert.*;
+
+import static com.github.tomakehurst.wiremock.client.WireMock.*;
+
+public class PayIDClientTest {
+    @Rule
+    public WireMockRule wireMockRule = new WireMockRule(wireMockConfig().dynamicHttpsPort());
```

Comment:
You will want to make the http port dynamic as well (wiremock starts listening on both).
`new WireMockRule(wireMockConfig().dynamicPort().dynamicHttpsPort());`

Reply:
PayID only works with HTTPS but I can add this for completeness.